src/firebase.js

```diff
@@ -0,0 +1,20 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import firebase from 'firebase/app';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import 'firebase/firestore';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import 'firebase/database';
```

Comment:
Remove the eslint disable lines

Reply:
When I remove it, appears an error just in this way works. @mujz